### PR TITLE
[V1] FIX AttributeMerger

### PR DESCRIPTION
**Pull request by pierry01**
## BEFORE - BUG IN MIX
image

## AFTER - MIX WORKING PROPERLY
image

### CODE REVIEW

File: test/rbui/select_test.rb [hunk 1]
```diff
@@ -29,5 +29,6 @@ def test_render_with_all_items
     end
 
     assert_match(/John/, output)
+    assert_match('name="NAME"', output)
```
Comment (pierry01) [inline test/rbui/select_test.rb:32]: BEFORE
assert_match('name="NAME NAME"', output) ✅ 

AFTER
assert_match('name="NAME NAME"', output) ❌

**Review by iseth — APPROVED** [review]
Thanks

**Comment by iseth** [comment]
Can you also make a new pr to merge this into main too.

**Comment by iseth** [comment]
Also make sure all the ci passes

**Review by cirdes — APPROVED** [review]

**Pull request merged**